Client/Frontend/Browser/Tabs/GridTabViewController.swift

```diff
@@ -228,6 +221,13 @@ class GridTabViewController: UIViewController, TabTrayViewDelegate, Themeable {
         flowlayout.invalidateLayout()
     }
 
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) {
+            self.focusItem()
+        }
```

Comment:
Typically introducing a delay is a last resort kind of fix to a problem. Is there a way to address the bug without a delay?

Reply:
I think this is the third regression of this issue, I did not found another solution to this problem.

Reply:
It's not ideal but if it works for now I think it's fine. We are going to be rewriting the tab tray logic completely using Redux in the near future so it should be a temporary band-aid.

Reply:
`⚠️ Note:` 
- Since we will get to it during tab tray logic, let's add a comment explaining the 0.1 delay
- 0.1 is fine although it can vary between older and newer device performance, worth asking QA to test this on older devices